style.css

```diff
@@ -383,6 +383,100 @@ textarea:hover {
 
 /* end-form */
 
+/* cancel button */
+.hide {
+  display: none !important;
+}
+
+.active {
+  display: flex;
+}
+
+/* popup window */
+#popup {
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  z-index: 999;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 110vh;
+  background-color: white;
+  overflow: auto;
+}
+
+#popup-image {
+  position: relative;
+  width: 90%;
+  margin: 1rem 1rem;
+  border-radius: 8px;
+}
+
+#primary-text {
+  margin-left: 1rem;
+  max-width: 60%;
+  max-height: 179px;
+  overflow-y: hidden;
+}
+
+#popup h2 {
+  color: #172b4d;
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  font-size: 32px;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+#popup div ul {
+  margin-top: 1rem;
+}
+
+#popup p {
+  margin: 0 1rem;
+  font-size: 16px;
+}
+
+.livelink {
+  margin-left: 0.5rem;
+  height: 20px;
+  width: 17.99px;
+}
+
+.linkbutton {
+  display: flex;
+  height: 48px;
+  font-size: large;
+}
+
+.button {
+  background-color: #2aa874;
+  align-items: center;
+  margin: 1rem auto;
+  font-family: Inter, sans-serif;
+  color: #d2d2d2;
+  border-radius: 5px;
+}
+
+.button-group {
+  display: flex;
+  flex-direction: row;
+}
+
+#cancel-btn2 {
+  background-image: url(images/Icon.png);
+  background-size: contain;
+  position: fixed;
+  top: 2.6rem;
+  right: 2.9rem;
+  width: 50px;
+  height: 50px;
+  background-color: transparent;
+  z-index: 1000;
+}
+
```

Comment:
- [ ] Good work changing the cross button on the desktop version. The mobile version however still has the same pixilated `cross button`. Please implement the requested change throughout the project 😄 
![cross](https://user-images.githubusercontent.com/99656687/231673520-05fcbfd6-4ed5-4f56-9351-07a5960f0cba.png)